mkosi.conf.d/20-fedora.conf

```diff
@@ -56,7 +56,7 @@ BuildPackages=
         glibc-static
         libcap-static
         pam-devel
-        pkgconfig # pkgconf shim to provide /usr/bin/pkg-config
+        /usr/bin/pkg-config
```

Comment:
Do the same change for centos?

Reply:
The config for centos didn't mention pkg-config, so I added it.